symbulate/distributions.py

```diff
@@ -618,16 +666,33 @@ def __init__(self,
                  sd1=1.0, sd2=1.0, corr=0.0,
                  var1=None, var2=None, cov=None):
 
-        if corr is not None and not (-1 <= corr < 1):
+        if corr is not None and not (-1 <= corr <= 1):
             raise Exception("Correlation must be "
                             "between -1 and 1.")
 
-        self.mean = [mean1, mean2]
+        if (isinstance(mean1, float)) and (isinstance(mean2, float)):    
```

Comment:
Why does it have to be a float? Why can't a user specify an integer? Scipy is okay with it, so why are you so strict?